### ISSUE TITLE
Carthage Support

### ISSUE THREAD

**Issue by lucasderraugh** [spotify/ios-sdk#415]
This is a relatively simple issue to address but having [Carthage](https://github.com/Carthage/Carthage) support would be extremely useful. Since you don't include the actual source for the SDK (not buildable via Xcode), could you include the framework as a zip in your releases? This way I could add the repo to a Cartfile and use the framework.

The following are instructions into allowing a framework to work on with a Github release:
>Archive prebuilt frameworks into one zip file

>Carthage can automatically use prebuilt frameworks, instead of building from scratch, if they are attached to a GitHub Release on your project’s repository.

>To offer prebuilt frameworks for a specific tag, the binaries for all supported platforms should be zipped up together into one archive, and that archive should be attached to a published Release corresponding to that tag. The attachment should include .framework in its name (e.g., ReactiveCocoa.framework.zip), to indicate to Carthage that it contains binaries.

Thanks!

**Comment by pandaxbacon** [comment]
+1

**Comment by brki** [comment]
I agree, it would be nice to be able to use the ios-sdk with Carthage.

**Comment by joelkraut** [comment]
Please fix this nearly year-old, trivial issue.

**Comment by mxcl** [comment]
I'm unimpressed. :/

**Comment by gemedet** [comment]
+1

**Comment by JakubMazur** [comment]
+1

**Comment by paperview** [comment]
+1

**Comment by pxpgraphics** [comment]
+1
